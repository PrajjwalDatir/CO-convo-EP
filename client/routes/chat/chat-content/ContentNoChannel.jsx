import React from "react";
import ContentContainer from "./ContentContainer";
import noChannelsImage from "#/assets/images/undraw_messaging_app.svg";

function ContentNoChannel() {
  return (
    <ContentContainer>
      <div className="relative flex flex-grow items-center overflow-hidden">
        {/* <img
          className="absolute opacity-0 select-none right-0"
          alt="Chat Icon"
          src={noChannelsImage}
        /> */}
        <div className="mx-12 mb-8 z-10">
          <h1
            className="text-5xl whitespace-normal leading-tight font-bold"
            style={{ fontFamily: "Raleway, sans-serif" }}
          >
            No chat selected
          </h1>
          <p className="text-gray-700 text-xl mt-4">
            To start chatting, join or create a group from the sidebar
          </p>
        </div>
      </div>
    </ContentContainer>
  );
}

export default ContentNoChannel;
