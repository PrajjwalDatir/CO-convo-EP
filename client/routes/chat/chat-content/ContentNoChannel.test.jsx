import React from 'react';
import { mount } from 'enzyme';
import ContentNoChannel from './ContentNoChannel';

describe('<ContentNoChannel />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<ContentNoChannel />);
    expect(wrapper).toMatchSnapshot();
  });
});
