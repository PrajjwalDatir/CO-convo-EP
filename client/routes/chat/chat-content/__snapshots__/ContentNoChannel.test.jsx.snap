// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentNoChannel /> should match snapshot 1`] = `
<ContentNoChannel>
  <ContentContainer
    direction="flex-row"
  >
    <div
      className="my-8 flex flex-grow bg-white rounded-lg shadow-xl flex-row"
    >
      <div
        className="relative flex flex-grow items-center overflow-hidden"
      >
        <img
          alt="Woman looking at phone with chat message icon"
          className="absolute opacity-25 select-none right-0"
          src="test-file-stub"
        />
        <div
          className="mx-12 mb-8 z-10"
        >
          <h1
            className="text-5xl whitespace-normal leading-tight font-bold"
            style={
              Object {
                "fontFamily": "Raleway, sans-serif",
              }
            }
          >
            No channel selected
          </h1>
          <p
            className="text-gray-700 text-xl mt-4"
          >
            To start chatting, join or create a channel from the sidebar
          </p>
        </div>
      </div>
    </div>
  </ContentContainer>
</ContentNoChannel>
`;
